modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v1/form_fields.rb

```diff
@@ -20,7 +20,7 @@ def veteran_dob
           'F[0].Page_1[0].DateSigned[0]'
         end
 
-        def mailing_address_number_and_street
+        def mailing_address
```

Comment:
I changed the name because the address object contains much more than just the number and street (quite different than HLR).